webapp/src/components/sidebar/ModelLibrary.vue

```diff
@@ -106,48 +88,29 @@
                   href="https://en.wikipedia.org/wiki/Indexed_Database_API"
                   target="_blank"
                 >
-                  IndexedDB</a></button>.<br>
+                  IndexedDB
+                </a>
+              </button>
+              .
+              <br />
             </span>
 
             <div class="flex items-center justify-center">
               <button
-                :class="buttonClass()"
+                class="px-4 py-2 space-x-4 transition-colors border rounded-md hover:text-slate-900 hover:border-slate-900 focus:outline-none focus:ring focus:ring-slate-900 focus:ring-offset-2 border-slate-900 text-slate-900"
```

Comment:
```suggestion
              class="flex items-center justify-center px-4 py-2 space-x-4 outline outline-1 outline-slate-300 rounded-md transition-colors duration-200 text-slate-600 hover:text-slate-800 hover:outline-slate-800 focus:outline-none focus:ring-1 focus:ring-slate-800"
```
Make the button inline (as it was before) and while we're at it also make the style consistent with the webapp design
Currently:
![Screenshot 2024-06-03 at 11 51 06](https://github.com/epfml/disco/assets/33122365/9ca44457-a042-4767-ace2-b5ad276fe026)
Suggested:
![Screenshot 2024-06-03 at 11 51 27](https://github.com/epfml/disco/assets/33122365/ca0edbb0-eb5c-40a6-b4cf-55c188c9bb3f)